app/assets/i18n/strings_ja.json

```diff
@@ -28,7 +28,7 @@
     "open": "開く",
     "queue": "順番待ち",
     "quickSave": "クイックセーブ",
-    "quickSaveFromFavorites": "\"Favorites\"からクイックセーブ",
+    "quickSaveFromFavorites": "「お気に入り」からクイックセーブ",
```

Comment:
This is only a random walk that checks if people had trouble understanding this line, so there’s no need to credit me as a Japanese translator.